obscuro-ethereum-interaction.md

```diff
@@ -36,27 +36,27 @@ In the usual case, a rollup from the canonical chain (see POBI protocol) is fina
 - Note 3: The period is inverse to the number of L2 nodes. It should be long enough to give honest participants the chance to react and publish in the face of aggressive censorship attempts against them, but short enough not to degrade the user experience. We estimate that once the network reaches a healthy number of nodes, we can reduce it to 50-100 blocks (~ 10 minutes).
 
 #### Rule 2 - The competing forks
-Assuming the period chosen at rule #1 is enough, the only possible write attack performed by an actor that could hack the TEE will manifest as multiple parallel forks at least two rollups deep. The reason for this is that all valid TEEs run the same attested code that will choose the same canonical chain from the rollups published in the L1 block presented as proof. If the Management contract notices multiple forks, the rule is that finality is suspended on all forks, thus, withdrawals are suspended. If one of the forks becomes inactive, the rule is that all rollups on the alive fork become final once a standard period of 1 day has passed from the last L1 block that contained a rollup published on the inactive branch.
+Assuming the period chosen at rule #1 is enough, the only possible write attack performed by an actor that could hack the TEE manifests as multiple parallel forks at least two rollups deep. This is because all valid TEEs run the same attested code that chooses the same canonical chain from the rollups published in the L1 block presented as proof. If the Management contract notices multiple forks, the rule is that finality is suspended on all forks; thus, withdrawals are suspended. Likewise, if one of the forks becomes inactive, the rule is that all rollups on the alive fork become final once a standard period of 1 day has passed from the last L1 block that contained a rollup published on the inactive branch.
```

Comment:
Capital C in contract

Not a fan of semi-colons, I think the comma was correct in the first place.